models/response.js

```diff
@@ -0,0 +1,14 @@
+var Sequelize = require('sequelize');
+var sequelizeConnection = require('../server');
+
+
+const Response = sequelizeConnection.define('response', {
+    Choice: {
+      type: Sequelize.STRING
+      },
+      where: {
+        QuestionID: question.id
+      }
+    });
```

Comment:
can't use 'where' in schemas, can only use it when querying models (like in API routes)